representatives/models.py

```diff
@@ -175,6 +175,11 @@ class Chamber(HashableModel):
 
     hashable_fields = ['name', 'country', 'abbreviation']
```

Comment:
Couldn't we get rid of hashable_fields and use the metaclass unique_together attribute in HashableModel.compute_hash ? That would avoid duplicating code.

(also is Meta.unique_together parsed by django-admin makemigrations ? or do we have to create the migration manually ?)